pkg/env/dev.go

```diff
@@ -43,6 +43,7 @@ func (c *conn) Read(b []byte) (int, error) {
 }
 
 type Dev interface {
+	Interface
```

Comment:
I don't really understand this change/why it's necessary.

Reply:
When gomock generates its mocks, it makes it of the interface. Because Dev's interface isn't a superset of env.Interface's, it only generates the one method, and isn't compliant with env.Interface. Our code doesn't run into this because the type coersion only checks env.Dev on things already typed as env.Interface.